Nautilus/Assets/Gadgets/ScanningGadget.cs

```diff
@@ -40,6 +40,16 @@ public class ScanningGadget : Gadget
     /// </summary>
     public TechCategory CategoryForPda { get; set; }
 
+    /// <summary>
+    /// Whether to add after or insert before the target, for sorting purposes.
+    /// </summary>
+    public bool PdaAppendAfter { get; set; }
+
+    /// <summary>
+    /// It will be added/inserted next to this item or at the end/beginning if not found.
```

Comment:
Just a nitpick with docs: what is "it" here? How about "The icon in the Blueprints tab will be added/inserted next to this entry, or at the end/beginning if not found." ?